crates/as/package.json

```diff
@@ -21,7 +21,7 @@
     "asbuild": "asc index.ts -b build/optimized.wasm --use abort=wasi_abort --debug"
```

Comment:
```suggestion
    "asbuild": "asc index.ts -o build/optimized.wasm --use abort=wasi_abort --debug"
```

Reply:
@MaxGraey Thanks for the suggestion! Applied!